src/pather.py

```diff
@@ -174,7 +177,16 @@ def __init__(self, screen: Screen, template_finder: TemplateFinder):
             (Location.A5_WP, Location.A5_NIHLATHAK_PORTAL): [6, 8, 9],
             (Location.A5_QUAL_KEHK, Location.A5_NIHLATHAK_PORTAL): [12, 11, 10, 6, 8, 9],
             (Location.A5_QUAL_KEHK, Location.A5_WP): [12, 11, 10, 6],
-            (Location.A5_LARZUK, Location.A5_QUAL_KEHK): [13, 14, 5, 6, 10, 11, 12],
+            
+            (Location.A5_TOWN_START, Location.A5_CAIN): [3, 4, 5, 6, 10, 11, 12],            
+            (Location.A5_STASH, Location.A5_CAIN): [5, 6, 10, 11, 12],
+            (Location.A5_CAIN, Location.A5_NIHLATHAK_PORTAL): [12, 11, 10, 6, 8, 9],
+            (Location.A5_CAIN, Location.A5_WP): [12, 11, 10, 6],
+            (Location.A5_WP, Location.A5_CAIN): [6, 10, 11, 12],
+            (Location.A5_CAIN, Location.A5_STASH): [12, 11, 10, 6, 5],
+            (Location.A5_LARZUK, Location.A5_CAIN): [13, 14, 5, 6, 10, 11, 12],
```

Comment:
good point. I have cleared them out but i DID have to add one to the stash,
[2021-12-30 07:44:43,631] ERROR      Don't know how to traverse from a5_qual_kehk to a5_stash

Reply:
Ah yeah. Because previously stashing is always done before resurrecting. So that was not needed. Now botty will go to chain, id, go to stash, go to resurrect. But that's fine.